documentation/app/mkdocs.yml

```
@@ -1,6 +1,7 @@
 site_name: 'Template Documentation'
 nav:
   - Overview: index.md
+  - requirements: template-requirements.md
```

```suggestion
  - Requirements: template-requirements.md
```
But honestly I prefer `Pre-requisites`